### PR TITLE
Pair: Manisha and Patrick {Tree: √, Binary: wip, Graph: nothing}

### PR DESCRIPTION
**Pull request by manishavi**
Tree test is done.
Binary Search Tree and Graph in progress

### CODE REVIEW

**Comment by manishavi** [comment]
Working on Binary

**Comment by mixelpixel** [comment]
merging Manisha's commits to my fork

File: src/graph.js [hunk 1]
```diff
@@ -3,6 +3,17 @@ class Graph {
   constructor() {
     this.graph = {};
   }
+  addNode(newNode, toNode) {
+    this.graph.push({
+    newNode,
```
Comment (seanchen1991) [inline src/graph.js:8]: Indent these two lines for better code readability.

**Review by seanchen1991 — APPROVED** [review]
It's awesome that you are digging into the code so much. Keep up the good work! :)

**Pull request closed without merging**